### ISSUE TITLE
Error on first start

### ISSUE THREAD

**Issue by idkw** [binarybabel/docker-xwiki#1]
Using the docker run command as stated in the readme won't work : 

```
docker run -p 8080:8080 -v $(pwd):/xwiki-data binarybabel/xwiki:v9.7

Unable to find image 'binarybabel/xwiki:v9.7' locally
v9.7: Pulling from binarybabel/xwiki

88286f41530e: Pull complete 
34b39302a4b1: Pull complete 
0ddbfafeeac3: Pull complete 
3e5c9c251aaa: Pull complete 
1fb1aca225b7: Pull complete 
f35021e2eab3: Pull complete 
ab5aa1d1e3d5: Pull complete 
b4567de8a67c: Pull complete 
Digest: sha256:cd47229f6f695cb27f0ce320715be893ba06c4fd2215f223cce41928bc9e89f5
Status: Downloaded newer image for binarybabel/xwiki:v9.7
Starting XWIKI-DOCKER ...
|
|    VERSION: 9.7
|    APP_DIR: /xwiki
|   DATA_DIR: /xwiki-data
|   JAVA_XMX: 1024m
| JETTY_PORT: 8080
|
sed: /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: No such file or directory
sed: /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: No such file or directory
XWIKI-DOCKER: Configured for filesystem attachments
./docker-entrypoint.sh: line 51: can't create /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: nonexistent directory
./docker-entrypoint.sh: line 51: can't create /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: nonexistent directory
./docker-entrypoint.sh: line 51: can't create /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: nonexistent directory
./docker-entrypoint.sh: line 51: can't create /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: nonexistent directory
./docker-entrypoint.sh: line 51: can't create /xwiki/webapps/xwiki/WEB-INF/xwiki.cfg: nonexistent directory
Error: Unable to access jarfile jetty/start.jar
```

**Comment by gurpal2000** [comment]
+1
Could it be that this is not the standalone version and requires tomcat or something?

**Comment by babelop** [comment]
Thank you for the bug-report. This is indeed a stand alone image. It is rebuilt automatically with the latest XWiki version and sometimes that breaks the build if they move files around.

This has been fixed here and republished as of the latest/v9.8.1 docker images. Please note however the broken v9.8.1 image was replaced, so you may need to do a forced pull if you are specifying a direct version number in your image reference.

I've also added a more verbose warning to startup script that will detect if required files are missing and point users directly to this bug tracker.

Thank you again for the heads up.
